Standalone/start-selenium-standalone.sh

```diff
@@ -11,6 +11,10 @@ if [ ! -z "$SE_OPTS" ]; then
   echo "Appending Selenium options: ${SE_OPTS}"
 fi
 
+if [ ! -z "$SE_DRIVER_OPTS" ]; then
+  echo "Appending Selenium Driver Service property: ${SE_DRIVER_OPTS}"
+fi
+
```

Comment:
This won't be used as we are always setting the browser driver before launching the Grid.